docs/ops/opset3.md

```diff
@@ -62,7 +62,7 @@ declared in `namespace opset3`.
 * [GroupConvolutionBackpropData](convolution/GroupConvolutionBackpropData_1.md)
 * [GRUCell](sequence/GRUCell_3.md)
 * [HardSigmoid](activation/HardSigmoid_1.md)
-* [Interpolate](image/Interpolate_1.md)
+* [Interpolate](image/Interpolate_3.md)
```

Comment:
As we discussed during the meeting we need to rename this operation to Intepolate_4 and move it to the docs/opst/opset4.md

Reply:
Fixed.